package.json

```diff
@@ -9,20 +9,23 @@
 		"build": "tsc",
 		"build-watch": "tsc -w",
 		"prepublishOnly": "npm run build",
-		"eslint": "eslint \"src/**/*.ts\" --max-warnings 0"
+		"eslint": "eslint .\\src\\"
 	},
 	"dependencies": {
-		"@types/node": "^20.8.0",
-		"@typescript-eslint/experimental-utils": "^5.62.0",
-		"typescript": "^5.2.2"
+		"@types/node": "^20.12.0",
+		"@typescript-eslint/utils": "^7.8.0",
+		"typescript": "^5.4.0"
 	},
 	"devDependencies": {
-		"@typescript-eslint/eslint-plugin": "^6.7.3",
-		"@typescript-eslint/parser": "^6.7.3",
-		"eslint": "^8.50.0",
-		"eslint-config-prettier": "^9.0.0",
-		"eslint-plugin-prettier": "^5.0.0",
-		"prettier": "^3.0.3"
+		"@typescript-eslint/eslint-plugin": "^7.8.0",
+		"@typescript-eslint/parser": "^7.8.0",
+		"eslint": "8.57.0",
```

Comment:
Style: npm uses `^` by default if just a version number is given, but adding it explicitly is easier to read and consistent with the other dependencies